core/controllers/creator_dashboard.py

```diff
@@ -164,12 +164,28 @@ def get(self):
         """Handles GET requests."""
 
         def _get_intro_card_color(category):
+            """Returns the intro card color according to the category.
+
+            Args:
+                category: str. The category of the exploration.
+
+            Returns:
+                str. The intro card color according to the category.
+            """
             return (
                 constants.CATEGORIES_TO_COLORS[category] if
                 category in constants.CATEGORIES_TO_COLORS else
                 constants.DEFAULT_COLOR)
 
         def _round_average_ratings(rating):
+            """Returns the rounded average rating of the exploration.
```

Comment:
```suggestion
            """Returns the rounded average rating to display on the creator dashboard.
```
I think this function is used to get the average rating of all explorations and collections created by the user. Could you check? Thanks!

Reply:
Oh, yes! I covers multiple explorations and collections. Updated!

Reply:
Done